### ISSUE TITLE
A training gap with codebase semantic-segmentation

### ISSUE THREAD

**Issue by yamengxi** [ekzhang/fastseg#7]
I trained the model MobileV3Small(LR-ASPP, F=128) using your [codebase](https://github.com/ekzhang/semantic-segmentation)(branch main)，but get mIOU with only 46.09. My training environment is:
python 3.7.6
pytorch1.5
GPU driver version 418.67
CUDA 10.1.243
cudnn 7.6.5
nccl 2.6.4,
openmpi 4.0.3,
gcc 5.4.0,
miniconda 4.8.2,
I start training with command `python -u -m runx.runx scripts/train_mobilev3small.yml -i &`, besides I used 8 Tesla V100 GPUS (the same as you mentioned in [here](https://github.com/ekzhang/fastseg#pretrained-models-and-metrics)) to train the model.
At first, the code can not run correctly (some errors like: mismatched tensor size), so I make some necessary modifications on your code (check my code [here](https://github.com/yamengxi/semantic-segmentation)(branch my_version)). After that, I didn't make any other modification (eg: learning rate, batch size,) and started training directly with command `python -u -m runx.runx scripts/train_mobilev3small.yml -i &`.
Why did I get a much lower result than you? Hope to get the answer, thank you very much!

**Comment by ekzhang** [comment]
Thank you for making those bug-fixes, as I am not able to run the code right now. 46.09 seems like a very low accuracy. I'm not sure what is going wrong. Can you maybe try training the version with F=256, increase epochs to 250, and see what happens?

**Comment by yamengxi** [comment]
I trained the model MobileV3Small(LR-ASPP, F=128) with max_epoch 300, and get a result with mIOU only **51.53**.
Now, I am training the model MobileV3Large(LR-ASPP, F=256) with max_epoch 300 and waiting for the result.

**Comment by yamengxi** [comment]
@ekzhang I trained the model MobileV3Large(LR-ASPP, F=256) with max_epoch 250 (as you said), and get a result with mIOU only **58.77**.
It is confusing.
